L1Trigger/TrackFindingTracklet/data/calcNumDTCLinks.txt

```diff
@@ -0,0 +1,432 @@
+0_2S_1	0
```

Comment:
This file is new. What is it? Can't these numbers be obtained from the Tracker cabling map?

Reply:
This file was copied from the previous default branch when I added back the data/ directory after rebasing:
https://github.com/cms-L1TK/cmssw/blob/f093f1b30f436c7323f132d7a0f51753ccf0ae3b/L1Trigger/TrackFindingTracklet/data/calcNumDTCLinks.txt

As far as I can tell though, it's not used anywhere, and I have removed it.